common/app/googleAuth/FilterExemptions.scala

```diff
@@ -1,17 +1,15 @@
-package http
+package googleAuth
 
 import com.gu.googleauth.FilterExemption
 
-object FilterExemptions {
+case class FilterExemptions(additionalUrls: String*) {
 
   lazy val loginExemption: FilterExemption = FilterExemption("/login")
   lazy val exemptions: Seq[FilterExemption] = List(
+    // Default
     FilterExemption("/oauth2callback"),
     FilterExemption("/assets"),
     FilterExemption("/favicon.ico"),
-    FilterExemption("/_healthcheck"),
-    FilterExemption("/2015-06-24-manifest.json"),
-    // the healthcheck url
-    FilterExemption("/world/2012/sep/11/barcelona-march-catalan-independence")
```

Comment:
This may be a silly question, but I would have expected that the [new healthcheck url](https://github.com/guardian/frontend/commit/68b44f3323955572b4043b443bae4bac9db2f4c0) to have to be exempt as well. Is this not the case?